DMPowerTools/Pages/Combat.razor

```diff
@@ -117,4 +125,14 @@
         }
         return selectedMonster.Actions.Count();
     }
+    public void InitiativeRoll()
+    {
+        Random random = new Random();
+        foreach (ActiveCreature a in ActiveCreatrues)
+        {
+            var roll = random.Next(1, 20);
```

Comment:
We don’t need it this time but I imagine since we have quite a few dice rolling concepts that are going to come into play, that a DiceRoll(er?) class is going to emerge quickly.

Alongside that, maybe Creature has that concept baked into it that does all the attribute and modifier calculations in it? Or a service?